edge/pkg/eventbus/dao/topicopr.go

```diff
@@ -0,0 +1,31 @@
+package dao
+
+import "k8s.io/klog/v2"
+
+func InsertSubTopics(msg string) error {
+	var err error
+	event := &SubTopics{
+		Topic: msg}
+	err = InsertTopics(event)
+	if err != nil {
+		klog.Infof("insert event failed")
```

Comment:
Incorrect log level, besides, raise error only works for most time, log here would generate similar and useless logs.